app/routes/__auth/user.server.ts

```diff
@@ -156,3 +161,47 @@ export async function refreshUser(refreshToken: string, session: Session) {
   await updateSession(parsedTokenSet, session)
   return parsedTokenSet.user
 }
+
+/**
+ * Gets another user from cognito
+ *
+ * @param sub - the sub of another user
+ * @returns a user if found, otherwise undefined
+ */
+export async function getOtherUser(sub: string) {
```

Comment:
This is a confusing name for this function, because it doesn't return something that has the same type as `getUser`. A better name would be `getCognitoUsernameForSub`. Also this function doesn't have much use outside of the endorsements server; can it go in endorsements.server.ts?

Reply:
Sounds good, moving it and the addUserToGroup function in the next commit